modules/setup/includes/functions.inc.php

```diff
@@ -129,11 +129,11 @@ function findString($filename) {
                 // Check if the line contains the string we're looking for, and print if it does
                 if(strpos($line, $search1) !== false){
                     $val = substr($line,9,2);
-                    echo $val;
+                    return $val;
```

Comment:
Ehhhmm @aethelwulffe here is what i do i have done a recursive read on the sites dir to look for all folders containing the sqlconfig file **first**. When i grab them i now do a read in each file to get the value of the `$config` variable. This permits me to know which site has a 1 or 0 so i know which installations is/are active or not. If cause it is no point asking someone to login with when the config variable has changed. With this set the dropdown would be implemented and you on select you would be redirected to `localhost/lh-ehr?site=<folder_name>`. Now in my force install method. You would be prompted all the available sites whether installed or not installed. Here the role of this is just to write a zero into the config variable and continue the process of installation normally